JS-week-2/event_application.js

```diff
@@ -0,0 +1,10 @@
+function weekdayCalc(days){
+    const weekdays = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+    let today = new Date();
+
+    today.setDate(today.getDate() + days);
+
+    console.log("The days is : "+weekdays[today.getDay()]);
+}
+
+weekdayCalc(8 );
```

Comment:
Try adding more cases like negative numbers or zero to see how your program behaves. I think it should handle it well, since you are using the `setDate` method which is really robust, but it's always good to have more test cases to test against so called edge-cases. 

There is a [nice blog post](https://medium.com/jeremy-gottfrieds-tech-blog/algorithms-common-edge-cases-in-js-ce35a2d47674) on handling edge cases, it's a bit more advanced but I figured it might be interesting to read.

Reply:
Hi @molefrog I have added the more cases, and code is working.